docs/en/04_Changelogs/4.0.0.md

```diff
@@ -941,6 +941,7 @@ specific functions.
   service name from the filename. Now the service name will either
   by the array key, or the `class` parameter value.
 * Uniqueness checks for `File.Name` is performed on write only (not in `setName()`)
+* Created `Resettable` interface to better declare objects which should be reset between tests.
```

Comment:
Add the following to the highlevel changelog summary:

 * Added a server requirement for the [php-intl](http://php.net/manual/en/book.intl.php) extension (shipped by default with most PHP distributions)
 * Replaced Zend_Date and Zend_Locale with the [php-intl](http://php.net/manual/en/book.intl.php) extension.
 * Consistently use CLDR date formats (rather than a mix of CLDR and date() formats)

Also, you need to update server-requirements.md